spec/operators-spec.coffee

```diff
@@ -913,6 +926,48 @@ describe "Operators", ->
         expect(vimState.getRegister('"').text).toBe '123'
         expect(editor.getCursorBufferPositions()).toEqual [[0, 0], [1, 2]]
 
+    describe "in a long file", ->
+      beforeEach ->
+        editor.setHeight(400)
+        editor.setLineHeightInPixels(10)
+        editor.setDefaultCharWidth(10)
+        text = ""
+        for i in [1..200]
+          text += "#{i}\n"
+        editor.setText(text)
+
+      describe "yanking many lines forward", ->
+        it "does not scroll the window", ->
+          editor.setCursorBufferPosition [40, 1]
+          top40 = editor.getScrollTop()
```

Comment:
A minor aesthetic point: I think it would fit in a bit better w/ our test suite if you renamed `top40` to `previousScrollTop` (and similarly for `top140` below).